posts/2021-03-22-announcing-the-lunatech-beginner-quarkus-course.adoc

```diff
@@ -0,0 +1,35 @@
+= Announcing the Lunatech Beginner Quarkus Course
+Neil Benn <neil.benn@lunatech.fr>
```

Comment:
it should be your GitHub handle ;)

Reply:
😆 I keep seeing this comment. I'll add it to the template so it reminds people.

Reply:
yea, all I did here was follow the `g8` template instructions, and the example it gave was Nicolas's email address:

```
[neilbenn@MacBook-Pro-de-Neil lunatech-blog (master)]$ g8 file://.
[info] resolving Giter8 0.13.1...

title [The title of you post]: Announcing the Lunatech Beginner Quarkus Course
date [2019-05-01]: 2021-03-22
author [Nicolas Leroux <nicolas@lunatech.com>]: Neil Benn <neil.benn@lunatech.fr>

Template applied in /Users/neilbenn/code/lunatech-blog/.
```